Cargo.toml

```diff
@@ -1,39 +1,41 @@
 [package]
-name = "curv"
-version = "0.7.1"
+name = "curv-kzen"
+version = "0.8.0-rc1"
 edition = "2018"
 authors = ["Omer Shlomovits"]
```

Comment:
please add your name here

Reply:
This field is being deprecated actually, authors are not displayed at crates.io and docs.rs anymore. But I don't mind joining you here 🙂

Reply:
done!